metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema_gen.py

```diff
@@ -55,6 +62,7 @@
     METADATA_EXTRACTION,
     PROFILING,
 )
+from datahub.metadata._urns.urn_defs import TagUrn
```

Comment:
simpler
```
from datahub.metadata.urns import TagUrn
```

Reply:
Ahh, nice catch, my ide tried to be too smart.